### ISSUE TITLE
Failing on fresh install and attempt to setup

### ISSUE THREAD

**Issue by lakshyaranganath** [semantic-release/semantic-release#523]
Steps to reproduce:
```
npm i -g semantic-release-cli
semantic-release-cli setup
```

What happens:
```
/Users/lakshya/.nvm/versions/node/v6.11.2/lib/node_modules/semantic-release-cli/src/index.js:32
module.exports = async function(argv) {
                       ^^^^^^^^

SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/lakshya/.nvm/versions/node/v6.11.2/lib/node_modules/semantic-release-cli/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
```

Versions:
node: 6.11.2
npm: 4.6.1

**Comment by pvdlg** [comment]
This is caused by https://github.com/babel/babel/pull/5487. It happened under certain circumstances based on how the dependencies ends up being installed (that depends of the npm your are using). It shouldn't happen with npm 5.

It's fix in semantic-release/cli#153

**Comment by lakshyaranganath** [comment]
Awesome! Thanks for the quick turnaround 👍 💯
